### PR TITLE
BaSP-MR-16: Update Postman.json

### PR DESCRIPTION
**Pull request by SantiH97**
paths updated on the .json postman file

### CODE REVIEW

**Review by marcohzo — APPROVED** [review]

**Comment by SantiH97** [comment]
Merge conflicts resolved, we should be good to go.

File: src/controllers/class.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 
-const Class = require('../models/Class');
+const Class = require('../models/class');
```
Comment (BareaMartin) [inline src/controllers/class.js:3]: For my understanding the team wanted the models to be with capital first letter. is this change intended?

**Review by BareaMartin — CHANGES_REQUESTED** [review]

File: src/controllers/class.js (hunk 1 above)
Comment (SantiH97) [reply src/controllers/class.js:3]: I know but the server was unable to run with those capitalized so I had to change them back in order to push

Comment (SantiH97) [reply src/controllers/class.js:3]: I'm guessing classes and subscriptions will change their respective pathings in their forthcoming PRs
